sdk/examples/how_tos/accounts_and_addresses/check_balance.rs

```diff
@@ -2,26 +2,44 @@
 // SPDX-License-Identifier: Apache-2.0
 
 //! In this example we sync the account and get the balance.
-//! Rename `.env.example` to `.env` first.
 //!
-//! `cargo run --release --all-features --example check_balance`
+//! Make sure that `example.stronghold` and `example.walletdb` already exist by
+//! running the `create_account` example!
+//!
+//! Rename `.env.example` to `.env` first, then run the command:
+//! ```sh
+//! cargo run --release --all-features --example check_balance
+//! ```
+
+use std::env::var;
 
 use iota_sdk::wallet::{Result, Wallet};
 
 #[tokio::main]
 async fn main() -> Result<()> {
-    // Create the wallet
-    let wallet = Wallet::builder().finish().await?;
+    // This example uses secrets in environment variables for simplicity which should not be done in production.
+    dotenvy::dotenv().ok();
 
-    // Get the account
+    let wallet = Wallet::builder()
+        .with_storage_path(&var("WALLET_DB_PATH").unwrap())
+        .finish()
+        .await?;
     let account = wallet.get_account("Alice").await?;
 
     // Sync and get the balance
-    let _balance = account.sync(None).await?;
+    let _ = account.sync(None).await?;
```

Comment:
I would not even do the `let _` then, it's confusing. And edit the comment above.

Reply:
we actually get the balance right after this line. Don't know this example didn't do it in one step already. Changed

Reply:
Done